node/src/protocol/triple.rs

```diff
@@ -9,6 +9,12 @@ use std::collections::hash_map::Entry;
 use std::collections::{HashMap, VecDeque};
 use std::sync::Arc;
 
+/// Maximum number of messsages that should be relayed when generating a stockpile of triples.
+pub const DEFAULT_MAX_MESSAGES: usize = 22500;
```

Comment:
Is this number determined empirically?

Reply:
I just chose 100 because I remember Michel saying that number was a good number for stockpiling but don't exactly remember where anymore. @itegulov would you happen to remember?